src/components/Filters/Filters.tsx

```diff
@@ -430,7 +431,7 @@ class FiltersInner extends Component<CombinedProps, State> {
         <div className={styles.Filters}>
           {filtersControlMarkup}
           {filtersContainerMarkup}
-          {tagsMarkup}
+          <span testID="tagsContainer">{tagsMarkup}</span>
```

Comment:
I would love suggestions for how to better target the `TagsWrapper` to test that it renders as a `VisuallyHidden` component. The test below that matches this change broke with the TextField change because it was looking for `VisuallyHidden` _anywhere_ in the Filters component which was quite brittle. 

I did try adding the testID to `TagsWrapper` directly and the test did pass, however Typescript doesn't like it due to the prop not existing on the `VisuallyHidden` component.

The bigger picture fix would probably be to refactor this large components into more easily testable pieces but that's out of the scope of this work.

Reply:
Maybe we could test the logic that uses `VisuallyHidden` instead?

https://github.com/Shopify/polaris-react/blob/46c2c4536979ae58bebc66505540725dae88c4ee/src/components/Filters/Filters.tsx#L359

So we could compare how many `VisuallyHidden` elements there are by changing `appliedFilters`?

Reply:
By that, do you mean diff the amount of `VisuallyHidden` elements before/after passing `appliedFilters`? 

My main concern with the original test was how brittle it was due to it checking the entirety of the `Filters` component for instances of `VisuallyHidden` when this logic only impacts a small part of the component. Narrowing down the change to when `appliedFilters` changes would definitely be a huge improvement, just a little concerned by how many different markup elements we're checking against which could result in false flags (either this test passing when it shouldn't if there are changes that could break this, or other false failures like this where the component is still working as intended).